test/acceptance/framework/k8s/deploy.go

```diff
@@ -133,15 +133,26 @@ func CheckStaticServerConnectionMultipleFailureMessages(
 // CheckStaticServerConnectionSuccessful is just like CheckStaticServerConnection
 // but it always expects a successful connection.
 func CheckStaticServerConnectionSuccessful(t *testing.T, options *k8s.KubectlOptions, deploymentName string, curlArgs ...string) {
+	t.Helper()
 	start := time.Now()
-	CheckStaticServerConnection(t, options, true, deploymentName, "", curlArgs...)
+	CheckStaticServerConnection(t, options, true, deploymentName, nil, curlArgs...)
 	logger.Logf(t, "Took %s to check if static server connection was successful", time.Since(start))
 }
 
 // CheckStaticServerConnectionSuccessful is just like CheckStaticServerConnection
 // but it always expects a failing connection with error "Empty reply from server."
 func CheckStaticServerConnectionFailing(t *testing.T, options *k8s.KubectlOptions, deploymentName string, curlArgs ...string) {
-	CheckStaticServerConnection(t, options, false, deploymentName, "curl: (52) Empty reply from server", curlArgs...)
+	t.Helper()
+	CheckStaticServerConnection(t,
+		options,
+		false,
+		deploymentName,
+		[]string{
+			"curl: (52) Empty reply from server",
+			"curl: (7) Failed to connect to static-server port 80: Connection refused",
+			"curl: (7) Failed to connect to static-server.ns1 port 80: Connection refused",
```

Comment:
Just curious, did you see these additional errors when testing out tproxy?

Reply:
Yeah, when with iptables we now get a connection refused. This is most likely because envoy itself will now reject the connection, whereas previously this error was coming from the upstream proxy that verifies intentions.